### PR TITLE
Forecast added and code cleanup

### PR DESCRIPTION
**Pull request by mkutny**
I generally cleaned up the code (replaced `var`'s, switched to 'strict mode') and added a forecast method.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -45,5 +55,15 @@ function MovingAverage (timespan) {
     return v
   }
 
+  ret.deviation =
+  function deviation() {
+    return d
+  }
+
+  ret.forecast =
```
Comment (pgte) [inline index.js]: Nice! Could you also add this new function to the README? Thanks so much!

**Comment by pgte** [comment]
The tests were failing, but I believe that it was because of ES6 support because of the node version.
Now only supports node v6 and 8.
If you rebase or merge from latest master, the tests should be passing.

**Comment by mkutny** [comment]
Hope it will go smoothly now.

**Comment by pgte** [comment]
@mkutny I believe there were some linting errors. You can run the linter using `npm run lint".

**Comment by pgte** [comment]
@mkutny thanks!

Landed on v1.0.0.

**Pull request merged**